config/sync/core.entity_view_display.eventinstance.default.default.yml

```diff
@@ -49,10 +49,10 @@ content:
       timezone_override: ''
       date_format: 'Y-m-d\TH:i:s'
       separator: '-'
-      one_day: 'd F Y'
-      one_month: 'd - {d} F Y'
-      several_months: 'd F - {d} {F} Y'
-      several_years: 'd F Y - {d} {F} {Y}'
+      one_day: 'j. F Y'
+      one_month: 'j. - {j}. F Y'
+      several_months: 'j. F - {j}. {F} Y'
```

Comment:
Please consider if there is a misplaced `.` here.

Should it be `several_months: 'j. F - {j.} {F} Y'`?

Reply:
Nope - it's correct.
`{j}` just means the end_date format.

I get the confusion though, but if I do `{j.}` it results in:

<img width="363" height="123" alt="Screenshot 2025-08-11 at 11 11 41" src="https://github.com/user-attachments/assets/4f37a917-01f3-4782-ab88-64bbeb1a8640" />

Reply:
OK. I think I got confused because some locations had `{j.}` and others had `{j}.`.

In any case it looks as if we use `{j}.` consistently now. That's good.